src/CasingOfString.md

```diff
@@ -92,3 +93,32 @@ func CheckCase(input string) string {
 	}
 }
 ```
+
+### Python
+```python
+
+def CheckCase(string):
```

Comment:
Getting the wrong result mate
![image](https://github.com/rahulrk-dev/snips/assets/47377566/ffc84d1e-ec6a-4274-9cfd-ede49ad12f6c)

Reply:
Spaces are not being handled